### DISCUSSION TITLE
I'm slowly learning the Program... but... having Level problems

### DISCUSSION THREAD

**Discussion by 15Tails** [Techokami/SonicWorldsNext#197] [Q&A] [answered]
I Duplicated the base zone with minor tweaks to learn how to make levels and use Godot and etc. but... when I go and try and change the Title card. it works... until I enter the normal Base Zone act 1 or Act 2... and it keeps my title cards changes...

How do I NOT have the engine do... that?

also I tried the godot discord server... they insisted I use labels instead... I don't even think that's how it works... I was also thinking of duplicating the script. but I had a strong feeling that wouldn't work either...

Edit: This also applies to Music as well... I want every other zone to have unique music as well. but that gets transferred over to other zones as well...

**Comment by 15Tails** [comment, marked as answer]
UGH. nevermind. I'm incredibly dumb I didn't think about checking the inspector tab for the title cards. I'm Sorry.

**Reply by NeonSRB2** [reply]
It's okay to not get everything immediately! I'll admit to my own fault of trying to force projectiles to the player collision flag.

"fuck around and find out"

**Reply by 15Tails** [reply]
Sorry for the incredibly late response. I picked up a really bad cold. I haven't been able to use the engine. but that's basically what I've been doing lol.

that's basically what I've been doing. "if it works, it works. however if it doesn't. fudge it. don't use it" lol.
